tests/features/permissions.spec.ts

```diff
@@ -60,6 +60,123 @@ export function testPermissions(): void {
       await dwn.close();
     });
 
+    it('should include record tags using the createRequest, createGrant and createRevocation if provided', async () => {
+      const alice = await TestDataGenerator.generateDidKeyPersona();
+
+      // createRequest with a protocol
+      const requestWrite = await PermissionsProtocol.createRequest({
+        signer      : Jws.createSigner(alice),
+        description : 'Requesting to write',
+        delegated   : false,
+        scope       : {
+          interface : DwnInterfaceName.Records,
+          method    : DwnMethodName.Write,
+          protocol  : 'https://example.com/protocol/test'
```

Comment:
Nit: considering assigning to a variable, too many occurrences... 🤣

Reply:
I modified it in some of them where it made sense, in the others i just kept it.

Reply:
Our wavelength are identical sometimes I take it for granted!